src/test/scala/testClass.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import uk.gov.hmrc.play.test.UnitSpec
+
+import scala.annotation.tailrec
+
+
+class test extends UnitSpec {
```

Comment:
Class name should be upper  `CamelCase`

http://docs.scala-lang.org/style/naming-conventions.html#classestraits